QEfficient/generation/text_generation_inference.py

```diff
@@ -625,6 +657,12 @@ def run_continuous_batching_decode(self, prompt_queue, generation_len):
 
                         self.session.set_buffers({"logits": logits_out_placeholder})
                         decode_pause_time += perf_counter() - start
+
+                        if self.prompt_to_lora_id_mapping_decode:
+                            decode_inputs["lora_ids"][decode_batch_id] = self.prompt_to_lora_id_mapping_decode[
+                                batch_id_map[decode_batch_id]
+                            ]
+
```

Comment:
can this go inside `_update_decode_input` method?

Reply:
Due to the necessity of referring to the correct id from `batch_id_map `(which is not in `_update_decode_input` but here), I need to include this line here.